api/RTCRtpStreamStats.json

```diff
@@ -0,0 +1,730 @@
+{
+  "api": {
+    "RTCRtpStreamStats": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCRtpStreamStats",
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": [
+            {
+              "version_added": "63"
+            },
+            {
+              "alternative_name": "RTCRTPStreamStats",
+              "version_added": "27"
+            }
+          ],
+          "firefox_android": [
+            {
+              "version_added": "63"
+            },
+            {
+              "alternative_name": "RTCRTPStreamStats",
+              "version_added": "27"
+            }
+          ],
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          },
+          "webview_android": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "codecId": {
```

Comment:
Added in [bug 902003](https://bugzilla.mozilla.org/show_bug.cgi?id=902003) for Firefox 27, as part of the original implementation of the interface: https://hg.mozilla.org/mozilla-central/rev/5372aea57cdb